### PR TITLE
Fixing start app with docker

### PR DESCRIPTION
**Pull request by dmarro89**
Injecting Logger and Configuration to the server.

Changes and edits:

* Creating of Server with Logger and Configuration injected

### CODE REVIEW

**Review by vdmitriyev — COMMENTED** [review]
Comments were provided

File: server/configuration.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func (c *ViperConfig) createDefaultConfigFile(cfgFile string) {
 
 	c.logger.Info("Creating default configuration file")
 
-	c.viper.SetDefault("server.host", "127.0.0.1")
+	c.viper.SetDefault("server.host", "0.0.0.0")
```
[the remark this answers is not in the document]
Comment (vdmitriyev) [reply server/configuration.go:51]: @dmarro89 Can you see this comment now?

[the remark this answers is not in the document]
Comment (dmarro89) [reply server/configuration.go:51]: Ok, I can revert it but if the docker image is not started with DARE_HOST=0.0.0.0 is not possible to interact with the database.
The only way to interact, in that case, is to enter into the bash of the container and send the http requests.

[the remark this answers is not in the document]
Comment (vdmitriyev) [reply server/configuration.go:51]: It is correct. That is why the option ```-e DARE_HOST="0.0.0.0"```` is explicitly in the run command of the ```dare-db-tls```  container. The same could be added to other ```dare-db``` docker run command as well

[the remark this answers is not in the document]
Comment (vdmitriyev) [reply server/configuration.go:51]: I would say that it is better to make something less convenient for an initial use, but more secure by design.

[the remark this answers is not in the document]
Comment (dmarro89) [reply server/configuration.go:51]: Ok, agree.
Tomorrow I'll revert it.

[the remark this answers is not in the document]
Comment (dmarro89) [reply server/configuration.go]: @vdmitriyev reverted.

**Review by vdmitriyev — APPROVED** [review]
Thanks for fast responses. I have added also a small note to readme.md on ```0.0.0.0``` in docker.

**Comment by dmarro89** [comment]
> Thanks for fast responses. I have added also a small note to readme.md on ```0.0.0.0``` in docker.
> 
> @dmarro89 PR could be merged 

Thanks

**Pull request merged**